datadog_lambda/handler.py

```diff
@@ -27,5 +51,11 @@ class HandlerError(Exception):
 
 (mod_name, handler_name) = parts
 modified_mod_name = modify_module_name(mod_name)
-handler_module = import_module(modified_mod_name)
-handler = datadog_lambda_wrapper(getattr(handler_module, handler_name))
+
+try:
+    start_time_ns = time.time_ns()
+    handler_module = import_module(modified_mod_name)
+    handler_func = getattr(handler_module, handler_name)
+    handler = datadog_lambda_wrapper(handler_func)
+except Exception as e:
+    handler = _ErrorOutsideHandlerDecorator(e, modified_mod_name, start_time_ns)
```

Comment:
Just want to confirm that we've checked start_time_ns reflects each invocation correctly?

Reply:
Good call out, I'll double check this

Reply:
This was a good catch, the start time / duration were being set incorrectly. I update the code to set the span start time based on initial time spent importing the handler + time spent in the call itself.